ui/overviewTab/overviewTab.pug

```diff
@@ -0,0 +1,107 @@
+// Overview section
+.overview.section
```

Comment:
This is an example of the above html converted to pug via an online tool.
I had to adjust it to get close to the same html, although there are some HTML differences.
For example, the `class` attribute will come before `name` attribute.

Some difference examples:

![image](https://user-images.githubusercontent.com/5629800/209893338-3493d18a-df99-4f88-84a5-c0fbe3f6f9c7.png)

```
+    <!-- Feats-->
     <div class="feat-overview no-add-or-modify-buttons">
       <div class="table-header">
         <div class="featextraordinaryletter">Extr</div>
         <div class="featname">Epic Feat</div>
       </div>
       <fieldset class="repeating_feat">
-        <span name="attr_featextraordinaryletter" class="featextraordinaryletter"></span>
-        <span name="attr_featname" class="featname"> </span>
+        
+        <span class="featextraordinaryletter" name="attr_featextraordinaryletter"></span>
+        <span class="featname" name="attr_featname"> </span>
       </fieldset>
     </div>
```